Makefile

```diff
@@ -28,5 +35,22 @@ output/sro_daily-matrix.md: templates/daily_matrix.mail.tmpl.md
              --template  $< \
              --output-file $@
 
+# NTO
+
+nto: output/nto_daily-matrix.html output/nto_daily-matrix.md
+
+output/nto_daily-matrix.html: templates/daily_matrix.tmpl.html
+	go run cmd/main.go --debug daily_matrix \
+	     --config-file examples/nto.yml \
+             --template  $< \
```

Comment:
`s/--template  $</--template $</`

Reply:
not sure what you mean here, I see twice the same thing ;-) (and Firefox's `ctrl-f` as well)

Reply:
hum, got it, double spaces